packages/x-components/src/components/suggestions/__tests__/highlight.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { mount, Wrapper } from '@vue/test-utils';
```

Comment:
Could you move this file next to the `highlight.vue` component file? Now it is under `suggestions` folder.